Icerm/Claude.lean

```diff
@@ -1,20 +1,30 @@
 import Lean
-import Icerm.API
 import Mathlib.Tactic
 open Lean Elab Command IO Syntax
 
+def anthropicApiKey : IO String := do 
```

Comment:
Does this mean you need to pass the API key with a flag to the server in VSCode? What shell environment does VSCode launch the server from?

Reply:
This works with a basic linux environment variable (I don't know how such things work in windows). In the latest commit I added support for setting the API key on the fly with the `#set_anthropic_api_key key` command.

Reply:
(Sorry for ignoring this yesterday) I think I'm missing something because my Lean server doesn't seem to pick up the environment variable from the shell. We can talk in person. I was hoping to avoid putting the API key into any checked-in files so that nobody would commit it accidentally, hence using an untracked `API.lean`. The env variable is definitely cleaner if it works.